users.go

```diff
@@ -260,6 +262,7 @@ type ModifyUserOptions struct {
 	ThemeID            *int    `url:"theme_id,omitempty" json:"theme_id,omitempty"`
 	PublicEmail        *string `url:"public_email,omitempty" json:"public_email,omitempty"`
 	CommitEmail        *string `url:"commit_email,omitempty" json:"commit_email,omitempty"`
+	Avatar             *string `url:"avatar,omitempty" json:"avatar,omitempty"`
```

Comment:
Could you maybe sort all the fields so they follow the order in the docs? It seems they updated it for this struct and they are now listed in alphabetical order.